Gibe.Navigation.Umbraco/INavigationElementFactory.cs

```diff
@@ -12,4 +12,19 @@ public interface INavigationElementFactory
 	{
 		INavigationElement Make(IPublishedContent content);
 	}
+
+	public class FakeNavigationElementFactory : INavigationElementFactory
```

Comment:
As this exists in a shared library and so this fake would be used to test new implementations of `INavigationProvider`, it may be worth making it a little more flexible. Maybe `FakeNavigationElementFactory(params Func<IPublishedContent, INavigationElement>[] foo)` or something?